sdk/search/Azure.ResourceManager.Search/tests/SearchManagementTestBase.cs

```diff
@@ -14,7 +14,7 @@ public class SearchManagementTestBase : ManagementRecordedTestBase<SearchManagem
     {
         protected ArmClient Client { get; private set; }
 
-        public AzureLocation DefaultLocation => AzureLocation.EastUS;
```

Comment:
@archerzz 
@ArcturusZhang is it possible to re-record tests using the eastus2euap endpoint? Please let me know